CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# Master
+
+## Breaking
+
+- `findRecursive` now returns an array instead of an `ArrayElement`.
```

Comment:
Also to note, `children` on array element returns array too. So IMO we should return array here for consistency.